src/components/Aside/index.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+import Item from "./Items";
+
+function Aside(props) {
```

Comment:
Do you import this component somewhere? If not - remove it

Reply:
Maybe i forgot to remove it.